R/GAMBLR.viz-package.R

```diff
@@ -0,0 +1,50 @@
+#' @keywords internal
```

Comment:
Curious what this file does?

Reply:
In the documentation of these functions (in `R/` folder), there is no `@import` or `@importFrom` field. To solve the dependency issue, I ran `usethis::use_import_from`, which automatically creates this `R/GAMBLR.viz-package.R` file with the `@importFrom` fields and add the dependency function to the `NAMESPACE` file.